tests/ceph_ansible/purge_dashboard.py

```diff
@@ -0,0 +1,56 @@
+""" Module to purge ceph dashboard"""
+import logging
+import re
+
+log = logging.getLogger(__name__)
+
+
+def run(**kw):
+    """
+
+    Args:
+       param kw:
+       - ceph_nodes: ceph node list representing a cluster.
+       - config: (optional)
+         - ansible-dir: path to ansible working directory, default is /usr/share/ceph-ansible.
+         - inventory: ansible inventory file, default is hosts.
+         - playbook-command: ansible playbook command string.
+
+
+    return: 0 on sucess, non-zero for failures
+    """
+    log.info("Running purge-dashboard test")
+    ceph_nodes = kw.get("ceph_nodes")
+    installer_node = None
+    ansible_dir = "/usr/share/ceph-ansible"
+    inventory = "hosts"
+    playbook_command = (
+        "infrastructure-playbooks/purge-dashboard.yml --extra-vars 'ireallymeanit=yes'"
+    )
+
+    playbook_regex = re.search("(purge-.*?\\.yml)(.*)", playbook_command)
+    playbook = playbook_regex.group(1)
+    playbook = f"infrastructure-playbooks/{playbook}"
+    playbook_options = playbook_regex.group(2)
+
+    # To get installer node in ceph nodes
+    for ceph in ceph_nodes:
+        if ceph.role == "installer":
+            installer_node = ceph
+            break
+
+    # command to be formed ansible-playbook -vvvv -i hosts infrastructure-playbooks/purge-dashboard.yml
+
+    cmd = f"cd {ansible_dir} ; ansible-playbook -vvvv -i {inventory} {playbook} {playbook_options}"
+
+    # executing the command to purge dashboard
+    out, err = installer_node.exec_command(cmd=cmd, long_running=True)
+    log.info("STDOUT:\n %s" % out)
```

Comment:
can we make use of f-strings?

Reply:
exec_command generate log records incrementally because of `long_running=True`. do we need to log STDOUT again?